contributing/code_of_conduct/index.rst

```diff
@@ -0,0 +1,92 @@
+size, disability, ethnicity, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+Our Standards
+-------------
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+Our Responsibilities
+--------------------
+
+:doc:`Enforcement team members </contributing/code_of_conduct/enforcement_team>`
+are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Enforcement team members have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+Scope
+-----
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by enforcement team members.
+
+Enforcement
+-----------
+
+Instances of abusive, harassing, or otherwise unacceptable behavior 
+:doc:`may be reported </contributing/code_of_conduct/reporting_guidelines>`
+by contacting the :doc:`enforcement team members </contributing/code_of_conduct/enforcement_team>`.
+All complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Enforcement team members who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
```

Comment:
This is too vague: `as determined by other members of the project's leadership.`  Who exactly is going to watch the enforcement team actions and which will be their tools to do that?

Reply:
We should update this section to be "core team" I would propose.

Reply:
OK. Don't forget to detail how the Core Team will oversee the Enforcement Team (I guess we'll need full private access to all decisions and reports to deem if there's been an inappropriate conduct by some Enforcement member).

Reply:
that is a good point .. how much transparency on the records is needed for oversight.
the enforcement team is likely going to receive formal training. furthermore incidents records may not even be shared with the entire enforcement team, depending on if they recuse themselves or people involved specifically wish they don't get involved.

as such its a bit tricky to give blanket full access to the records to the core team. but I very much acknowledge that in case over reach is suspected, access the the records may be critical to make a final judgement by the core team.

so this seems like an important topic to also cover within the process.

/cc @Nyholm @michaelcullum @egircys